changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 20th October 2020 12:00 UTC
+[Error codes](general-remarks.md#error) list extended. New error codes are from 9 to 14.
```

Comment:
@Ksenia-Galtsova @patrikmiksovsky95  Please reformat this file to follow same conventions as connector API changelog:
- blank lines below headings
- change log items as a list items